app/javascript/src/components/Dashboard/Tasks/index.jsx

```diff
@@ -1,18 +1,20 @@
 import React, { useState, useEffect } from "react";
-import { Button, PageLoader } from "neetoui";
+import { Button, PageLoader, Toastr } from "neetoui";
 import EmptyState from "components/Common/EmptyState";
 import EmptyNotesListImage from "images/EmptyNotesList";
 import { Header, SubHeader } from "neetoui/layouts";
-import TaskList from "./TaskList";
+import TaskList from "./taskList";
 import NewTaskPane from "./NewTaskPane";
 import DeleteTaskAlert from "./DeleteTaskAlert";
+import moment from "moment";
```

Comment:
Please use `dayjs` over moment. 
And please sort the import statements accordingly.